rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -306,10 +306,38 @@
       "description": "Document is harvested or not, will disable record edition or similar.",
       "type": "boolean"
     },
-    "fiction": {
-      "title": "Fiction",
-      "description": "Document is fiction.",
-      "type": "boolean",
+    "fiction_statement": {
+      "title": "Fiction statement",
+      "description": "Fiction statement.",
```

Comment:
description: Defines whether the document is fiction or non-fiction (documentary). If irrelevant (image, object, score, etc.), use "unspecified".